.travis.yml

```diff
@@ -24,11 +37,12 @@ before_script:
   # - mysql -e 'create database sabredav'
   - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini  
   # - composer self-update
-  - composer update --prefer-source $LOWEST_DEPS
```

Comment:
IMO `--prefer-source` does not do anything useful here. We removed it in other repos.